diff --git a/dist/changesets-cli.cjs.js b/dist/changesets-cli.cjs.js
index 0f03e31068a5a7f77ef03ee97684d7f4d5136367..ceffa11754f414b475c948dc280b233cabce91cb 100644
--- a/dist/changesets-cli.cjs.js
+++ b/dist/changesets-cli.cjs.js
@@ -762,7 +762,9 @@ async function internalPublish(packageJson, opts, twoFactorState) {
   // Due to a super annoying issue in yarn, we have to manually override this env variable
   // See: https://github.com/yarnpkg/yarn/issues/2935#issuecomment-355292633
   const envOverride = {
-    [scope ? `npm_config_${scope}:registry` : "npm_config_registry"]: registry
+    [scope ? `npm_config_${scope}:registry` : "npm_config_registry"]: registry,
+    // Don't pass through any npm auth tokens so that we use trusted publishers
+    NODE_AUTH_TOKEN: "",
   };
   let {
     code,
